### PR TITLE
python.pkgs.feedgen: init at 0.5.1

### PR DESCRIPTION
**Pull request by casey**
###### Motivation for this change

Needed the feedgen package for development. First nixpkg, so I'm sure I messed something up.

###### Things done

Added the python feedgen. Used name `feedgen` since that's what it's called on pypi.

- [x] Tested using sandboxing
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Linux
- [N/A] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [N/A] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [N/A] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/feedgen/default.nix
Comment (lheckemann) [inline pkgs/development/python-modules/feedgen/default.nix]: You seem to be taking python as a function argument in spite of never using it in the expression. This won't affect evaluation (unless python goes missing), but is probably not desirable anyway :)

**Review by lheckemann — APPROVED** [review]
Looks pretty sensible overall.

File: pkgs/development/python-modules/feedgen/default.nix
Comment (adisbladis) [inline pkgs/development/python-modules/feedgen/default.nix]: Should not point to specific version

File: pkgs/development/python-modules/feedgen/default.nix
Comment (casey) [inline pkgs/development/python-modules/feedgen/default.nix]: Whoops, removed.

File: pkgs/development/python-modules/feedgen/default.nix
Comment (casey) [inline pkgs/development/python-modules/feedgen/default.nix]: Sweet, fixed

**Comment by casey** [comment]
I force pushed new commits. Do y'all generally prefer modifications to a PR to be in the form of force-pushed changes or add on diffs to be squashed later?

File: pkgs/development/python-modules/feedgen/default.nix
Comment (adisbladis) [inline pkgs/development/python-modules/feedgen/default.nix]: Could be simplified as:
`license = with licenses; [ bsd2 lgpl3 ];`

**Review by FRidh — APPROVED** [review]
Looks good to me. Before I merge, are you willing to maintain this package?

**Comment by casey** [comment]
@FRidh Yup, I'd be happy to. Should I add myself to maintainers.nix in this commit or a separate one?

**Comment by FRidh** [comment]
@casey you can do that in the same commit. And regarding your earlier question: it doesn't really matter. I personally prefer when its always in a state that it can go in right away.

**Comment by casey** [comment]
@FRidh Cool, added myself as a maintainer.

**Comment by casey** [comment]
Thanks all for reviewing!

**Pull request merged**
